### DISCUSSION TITLE
GitHub "act as me" permission?

### DISCUSSION THREAD

**Discussion by jjerhub-guest** [okteto/okteto#3429] [Q&A]
Why does your app require permission to act as me on GitHub? Isnt there some more granular "read repo"  permission?

**Comment by jLopezbarb** [comment]
Our GitHub App only act as a user to list repositories or branches.  When using a GitHub App to authenticate, you cannot choose the permissions. As Github states in their [docs](https://docs.github.com/en/apps/using-github-apps/authorizing-github-apps):
```
This ability to act on your behalf is limited to the GitHub resources where both you and the GitHub App have access. In some cases, however, the application may never make any changes on your behalf
```

Then, when managing the App, you can configure the list of repositories the GitHub App will be able to access.
